AnalyticsServiceStorageQueueInboundProcessorAdapter/function.json

```
@@ -0,0 +1,18 @@
+{
+    "bindings": [
+      {
+        "type": "queueTrigger",
+        "direction": "in",
+        "name": "serviceFailure",
+        "queueName": "%SERVICES_FAILURE_QUEUE_NAME%",
+        "connection":"INTERNAL_STORAGE_CONNECTION_STRING"
+      }
+    ],
+    "retry": {
+      "strategy": "exponentialBackoff",
+      "maxRetryCount": 5,
+      "minimumInterval": "00:00:05",
+      "maximumInterval": "00:30:00"
+    },
+    "scriptFile": "../dist/AnalyticsMessageStatusFromQueue/index.js"
```

Ditto.
I wonder if it would be useful to add some integration tests.

```suggestion
    "scriptFile": "../dist/AnalyticsServiceStorageQueueInboundProcessorAdapter/index.js"
```

---

I've done some try with IT, but it's not possible to use local EH (trigger are not complient with kafka) and cosmos db emuator seems to not work properly with change feed trigger.